code/unit_tests/recipe_tests.dm

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Unit Tests for various recipes.
+ *
+ */
+
+#define SUCCESS 1
+#define FAILURE 0
```

Comment:
You defined these but did not use them. Probably they should just be defined for all unit tests rather than in each individual one, since a bunch of them use it.